.vscode/settings.json

```
@@ -150,7 +150,7 @@
     "quickfix.biome": "explicit",
     "source.organizeImports.biome": "explicit"
   },
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.defaultFormatter": "biomejs.biome",
```

```suggestion
  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

default formatter を biome にすると yaml とか保存したときにエラー出るので prettier のままでしゃす